### PR TITLE
Tooltip + Dropdown fix

### LINKED ISSUE #32: Tooltip Component

### LINKED ISSUE #38: Tooltip + Dropdown alignment
Fix Tooltip and Dropdown alignment param by adding translate calc for it

### PR DESCRIPTION
**Pull request by nicosammito**
Close #32
Close #38

### CODE REVIEW

**Comment by nicosammito** [comment]
@Taucher2003 error with your pipeline

File: src/components/button/Button.stories.tsx [hunk 1]
```diff
@@ -88,4 +89,27 @@ export const ButtonGroups: ButtonGroupStory = {
...
+
+export const ButtonImage = () => {
+    return <ButtonGroup>
+        {
+            ["Pictor", "Sculptor", "Reticulum"].map(value => {
+                return <Tooltip>
+                    <Tooltip.Trigger>
+                        <Button variant={"primary"}>
+                            <Button.Icon>
+                                <img width={32}
```
Comment (Taucher2003) [inline src/components/button/Button.stories.tsx:102]: Missing `alt` attribute (this also causes the snapshot update pipeline to fail due to failing accessibility tests)

You can trigger the pipeline again after this is fixed.

File: src/components/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+                menu.style.transform = `translate(${calculatedPosition.x}px,${alignmentY}px)`
+            }
+        }
+
+        //calculate right position for the menu and set this as new attribute
+        calculatePosition()
+
+        //outside clicks that disables the dropdown
+
```
Comment (Taucher2003) [inline src/components/tooltip/Tooltip.tsx:82]: Is the code for the comment missing? 😆

Comment (nicosammito) [reply src/components/tooltip/Tooltip.tsx]: is handled over css instead of js. Different to Dropdown

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**